### DISCUSSION TITLE
Full body Avatar

### DISCUSSION THREAD

**Discussion by Kanishk-Malick** [mozilla/hubs#2819] [General Questions & Discussion] [answered]
Is it possible to upload full body avatar on hubs ? If yes, please tell how ?

**Comment by cvanmeurs** [comment]
There's nothing stopping anyone from putting a full-body avatar on Hubs, however there's currently no system in place to trigger a walk cycle when you move around, so your character will still appear to be floating when you move.

**Comment by emclaren** [comment, marked as answer]
Hi there, there is actually an earlier discussion thread about full body avatars. I'm going to lock this discussion but feel free to continue in the other thread: 

https://github.com/mozilla/hubs/discussions/2718
